python/tests/test_tables.py

```diff
@@ -3607,6 +3622,13 @@ def test_shuffled_tables(self):
             assert tables2.sites.num_rows == 0
             assert tables2.mutations.num_rows == 0
 
+    def test_doesnt_reorder_individuals(self):
+        tables = wf.wf_sim(N=5, ngens=5, num_pops=2, seed=123, record_individuals=True)
```

Comment:
Record individuals is now the default since #1238

Reply:
Thought so, but I didn't see it... oh, I need to rebase.